packages/hdwallet-native/src/mayachain.ts

```diff
@@ -0,0 +1,100 @@
+import type { StdTx } from "@cosmjs/amino";
+import type { SignerData } from "@cosmjs/stargate";
+import * as core from "@shapeshiftoss/hdwallet-core";
+import * as bech32 from "bech32";
+import CryptoJS from "crypto-js";
+import PLazy from "p-lazy";
+
+import * as Isolation from "./crypto/isolation";
+import { NativeHDWalletBase } from "./native";
+import * as util from "./util";
+
+const MAYA_CHAIN = "mayachain-mainnet-v1";
+
+const protoTxBuilder = PLazy.from(() => import("@keepkey/proto-tx-builder"));
```

Comment:
Same reason as above.
```suggestion
const protoTxBuilder = PLazy.from(() => import("@shapeshiftoss/proto-tx-builder"));
```